docker-compose.yml

```diff
@@ -735,7 +735,7 @@ services:
     ports:
       - 8088:8088
     environment:
-      CUB_CLASSPATH: '/usr/share/java/ksql-server/*:/usr/share/java/cp-base-new/*'
+      CUB_CLASSPATH: '/usr/share/java/ksqldb-server/*:/usr/share/java/cp-base-new/*'
```

Comment:
I think internally in this image, the jars are still in `/usr/share/java/ksql-server`

Reply:
Ack and reverted